pkg/dockerutil/dockerutils.go

```diff
@@ -331,6 +332,53 @@ func CheckDockerVersion(versionConstraint string) error {
 	return nil
 }
 
+// CheckDockerCompose determines if docker-compose is present and executable on the host system. This
+// relies on docker-compose being somewhere in the user's $PATH.
+func CheckDockerCompose(versionConstraint string) error {
+	executableName := "docker-compose"
+	if runtime.GOOS == "Windows" {
+		executableName = "docker-compose.exe"
+	}
+
+	path, err := exec.LookPath(executableName)
+	if err != nil {
+		return fmt.Errorf("no docker-compose")
```

Comment:
I know this is copied from the docker one, but I don't like either one. Maybe we could improve both with a more informative response? Like "docker-compose not found in environment" or something like that?

Reply:
This is just for internal use. The actual message shown to the user is in `cmd/ddev/cmd/root.go`. Do you want to change the internal string? Or were you just concerned about what the user would see?

Reply:
Yeah, it's OK. This isn't very good golang example code for you though :) If using it this way, we should probably be doing a typed error or whatever. In most cases we try to do error wrapping, where you can tell the lineage of the error when we print something out.

Reply:
Oh, interesting. Is there somewhere that does that in the ddev codebase that I should be looking at as an example?

Reply:
Mostly we do the textual wrapping, you'll see it almost everywhere there's a `return fmt.Errorf()`, but the caller will then *report* the cause within a wrapping message.